sdk/react/panels/AudioAndCCSelectionPanel.js

```diff
@@ -56,13 +64,20 @@ var AudioAndCCSelectionPanel = React.createClass({
   },
 
   onAudioTrackSelected: function(name) {
-    if (this.props.selectedAudioTrackTitle !== name) {
-      this.props.onSelectAudioTrack(name);
+    let localizedTitleForUndefinedLanguage = Utils.localizedString(this.props.config.locale, stringConstants.undefinedLanguageTitle, this.props.config.localizableStrings);
+    let localizedTitleForNoLinguisticContent = Utils.localizedString(this.props.config.locale, stringConstants.noLinguisticContentTitle, this.props.config.localizableStrings);
+    var originalName = name;
```

Comment:
I think you have the naming of these two variable backwards. If we edit the name, it's no longer the original name. I would rename the param that's passed in to be 'originalName', and the new localized one to be 'name'.

Reply:
Disregard the previous statement. I got the parameters mixed up.